docs/source/tutorials/token_classification.ipynb

```diff
@@ -0,0 +1,730 @@
+    "- PER (person entity)\n",
+    "- ORG (organization entity)\n",
+    "- MISC (miscellaneous other type of entity)\n",
+    "- O (other type of word that does not correspond to an entity)\n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:** \n",
+    "- Identify potential token label issues using cleanlab's `token_classification.filter.find_label_issues` method. \n",
+    "- Rank sentences using cleanlab's `token_classification.rank.get_label_quality_scores` method. \n",
+    "- TODO: (Clean Learning) Train a more robust model by removing problematic sentences. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1da020bc",
+   "metadata": {},
+   "source": [
+    "## 1. Install the required dependencies \n",
+    "\n",
+    "You can use `pip` to install all packages required for this tutorial as follows: \n",
+    "\n",
+    "    !pip install cleanlab "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ae8a08e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget -nc https://data.deepai.org/conll2003.zip && mkdir data \n",
+    "!unzip conll2003.zip -d data/ && rm conll2003.zip \n",
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/TokenClassification/pred_probs.npz' "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "439b0305",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Package installation (hidden on docs website).\n",
+    "# Package versions used: cleanlab==2.0.0 numpy==1.16.6 \n",
+    "# ericwang/cleanlab -b token_classification for now \n",
+    "\n",
+    "dependencies = [\"cleanlab\"]\n",
+    "\n",
+    "if \"google.colab\" in str(get_ipython()):  # Check if it's running in Google Colab\n",
+    "    %pip install cleanlab  # for colab\n",
+    "    cmd = ' '.join([dep for dep in dependencies if dep != \"cleanlab\"])\n",
+    "    %pip install $cmd\n",
+    "else:\n",
+    "    missing_dependencies = []\n",
+    "    for dependency in dependencies:\n",
+    "        try:\n",
+    "            __import__(dependency)\n",
+    "        except ImportError:\n",
+    "            missing_dependencies.append(dependency)\n",
+    "\n",
+    "    if len(missing_dependencies) > 0:\n",
+    "        print(\"Missing required dependencies:\")\n",
+    "        print(*missing_dependencies, sep=\", \")\n",
+    "        print(\"\\nPlease install them before running the rest of this notebook.\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "a1349304",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from cleanlab.token_classification.filter import find_label_issues \n",
+    "from cleanlab.token_classification.rank import get_label_quality_scores, issues_from_scores \n",
+    "from cleanlab.internal.token_classification_utils import * \n",
+    "from cleanlab.token_classification.summary import * "
```

Comment:
`import *` is discouraged in python code (has many potential downsides), so let's import all the things we need explicitly (even if it's a long list)